index.js

```diff
@@ -17,4 +19,27 @@ function Namely(options) {
     }
 }
 
+Namely.prototype.send = function (path, opts, cb) {
+    path = path + '.json';
+    opts = opts || {};
+
+    opts.headers = {
+        authorization: 'Bearer ' + this.accessToken
+    };
+
+    opts.json = true;
```

Comment:
If you find yourself always setting certain options you can use `require('request').defaults(...)` above to set them everywhere..